vaadin-grid.html

```diff
@@ -6,6 +6,19 @@
 
 <link rel='import' href='../polymer/polymer.html'>
 <script src="vaadin-grid.min.js"></script>
+<script>
+if (!window.vaadin) {
```

Comment:
Well we could be interested sometime in fact I debugged vaadin-grid in SDM with other GWT apps running SDM but never pure vaadin ones.
Anyway, you are right this is a very corner case.